packages/snap-tracker/src/Tracker.ts

```diff
@@ -211,6 +212,45 @@ export class Tracker {
 			return beaconEvent;
 		},
 
+		error: (data: TrackErrorEvent, siteId?: string): BeaconEvent | undefined => {
+			if (!data?.stack && !data?.message) {
+				// no console log
+				return;
+			}
+			let context = this.context;
+			if (siteId) {
+				context = deepmerge(context, {
+					context: {
+						website: {
+							trackingCode: siteId,
+						},
+					},
+				});
+			}
+			const payload = {
+				type: BeaconType.ERROR,
+				category: BeaconCategory.METRICS,
```

Comment:
I think `category` here is meant to provide a way to quickly distinguish various error types. I'm not sure we'd have any types here, but we'd probably want to use something like `searchspring.js` as the category root, and then be able to add to that as needed in the future. Eg: `searchspring.js.type` or `searchspring.js.syntax` or `searchspring.js.runtime` etc...

Reply:
Since we need the type to remain the same across any events we send in the future, I'm renamed `type` to be `BeaconType.METRIC` with a value of `metric`
And rename the category to `BeaconCategory.RUNTIME` with a value of `searchspring.js.runtime`

Reply:
Do we want type to be 'error' since that is likely the DB we will be creating?